README.md

```diff
@@ -1,50 +1,63 @@
+## What are the Components of the IDE?
 
-Let's spend a minute studying the Learn IDE user interface. You'll notice that it's composed of three sections: (1) a file browser (2) a text editor and (3) the console which you'll see in the image below. 
+Let's spend a minute studying the Learn IDE user interface. You'll notice that it's composed of three sections: (1) a file browser (2) a text editor and (3) the console which you'll see in the image below.
 
 ![IDE](https://s3.amazonaws.com/learn-verified/Screen+Shot+2016-04-15+at+3.48.17+PM.png)
 
 ### IDE File Browser
 
 ![IDEFileBrowser](https://s3.amazonaws.com/learn-verified/ILE-BrowserFile.png)
 
-The left side of your IDE application houses a simple file browser. It works just like a file browser on any computer: you can navigate a tree of folders and files to find the file you're looking for. Opening a file will open its contents in the text editor on the right. This is often the most convenient way to see the structure of a file tree and find a file quickly.
+The left side of your IDE application houses a simple file browser. It works just like a file browser on any computer: you can navigate a tree of folders and files to find the file you're looking for. Opening a file will open its contents in the text editor on the right. This is often the most convenient way to see the structure of a file tree and find a file quickly. You can open a file by double-clicking the file name.
 
-Reminder: the files you can see and access here are those that exist on the server you're connected to via the Learn IDE, not those on your local computer. 
+Reminder: the files you can see and access here are those that exist on the server you're connected to via the Learn IDE, not those on your local computer.
 
-### IDE Text Editor 
+### IDE Text Editor
 
 ![IDETextEditor](https://s3.amazonaws.com/learn-verified/ILE-TextEditor.png)
 
 The text editor is the heart of the IDE. It's where you'll write code, and where you'll be spending most of your time. Again, we built the IDE on top of Atom, a free open-source text editor by GitHub. One of the nifty things about text editors that have been optimized for programming is that as you type, your code will highlight in ways that make it easier to discern the structure of your program. You'll notice this when you start writing your own code.  
 
-When you click on a file in your file browser, it will open up in the text editor portion of the IDE and you can go ahead and edit right there. 
+When you click on a file in your file browser, it will open up in the text editor portion of the IDE and you can go ahead and edit right there.
+
+Hint: Don't forget to save your work often so you don't lose it! A good rule of thumb is to save and commit your code every 3-5 lines. You will learn more about committing your code to Github in a few lessons, but for now just save your work by going to File -> Save, command-s (Mac), or control-s (Windows).
```

Comment:
Put in a reminder here about "hey, uh, if you don't see a change (test/ viewing it with httpserver) it might be because you didn't save?